src/node.cc

```diff
@@ -4792,11 +4792,8 @@ inline int Start_TTDReplay(Isolate* isolate, void* isolate_context,
   Environment env(isolate_data, context);
   env.Start(argc, argv, exec_argc, exec_argv, v8_is_profiling);
 
-  bool debug_enabled =
-    debug_options.debugger_enabled() || debug_options.inspector_enabled();
-
   // Start debug agent when argv has --debug
-  if (debug_enabled)
+  if (debug_options.debugger_enabled() || debug_options.inspector_enabled())
```

Comment:
I am planning to do it in a day or 2. So yeah, consider it as prep work.

---
In reply to: [116625440](https://github.com/nodejs/node-chakracore/pull/244#discussion_r116625440) [](ancestors = 116625440)